packages/manager/apps/telecom/src/app/telecom/telephony/line/phone/order/telephony-line-phone-order.html

```diff
@@ -0,0 +1,187 @@
+            data-tuc-section-back-link-title="{{ 'telephony_group_line_phone_section_back_link' | translate }}"
+        >
+        </tuc-section-back-link>
+        <h1
+            data-ng-if="!$ctrl.phone"
+            data-translate="telephony_line_phone_order_title"
+        ></h1>
+        <h1
+            data-ng-if="$ctrl.phone"
+            data-translate="telephony_line_phone_order_change_title"
+        ></h1>
+    </header>
+
+    <div class="telecom-section-content">
+        <tuc-toast-message></tuc-toast-message>
+
+        <div class="text-center" data-ng-if="$ctrl.isLoading">
+            <oui-spinner></oui-spinner>
+        </div>
+
+        <div data-ng-if="!$ctrl.isLoading && $ctrl.hasPendingOfferTasks">
+            <!-- PENDING OFFER TASKS WARNING -->
+            <section class="col-xs-12 col-md-6 widget-presentation">
+                <header class="widget-presentation-header">
+                    <h2
+                        class="widget-presentation-title"
+                        data-translate="telephony_line_phone_order_order_title"
+                    ></h2>
+                </header>
+                <div
+                    class="alert alert-warning"
+                    role="alert"
+                    data-translate="telephony_line_phone_pending_task_warning"
+                ></div>
+            </section>
+        </div>
+
+        <div data-ng-if="!$ctrl.isLoading && !$ctrl.hasPendingOfferTasks">
+            <div data-ng-if="!$ctrl.line.isAttachedToOtherLinesPhone">
+                <!-- ORDER -->
+                <div>
+                    <section
+                        class="widget-presentation"
+                        data-ng-if="$ctrl.order.orderURL"
+                    >
+                        <header class="widget-presentation-header">
+                            <h2
+                                class="widget-presentation-title"
+                                data-translate="telephony_line_phone_order_order_title"
+                            ></h2>
+                        </header>
+                        <div
+                            class="alert alert-success"
+                            role="alert"
+                            data-translate="telephony_line_phone_order_success2"
+                        ></div>
+                        <p>
+                            <a
+                                target="_blank"
+                                class="btn btn-primary"
+                                rel="noopener"
+                                data-ng-href="{{:: $ctrl.order.orderURL }}"
+                            >
+                                <span
+                                    data-translate="telephony_line_phone_order_url"
+                                ></span>
+                                <i
+                                    class="fa fa-external-link"
+                                    aria-hidden="true"
```

Comment:
Please add a text alternative for the users with screen readers, like this: https://github.com/ovh/manager/blob/master/packages/manager/modules/support/src/tickets/new-ticket/new-ticket.html#L57